.github/workflows/release-swift-toolchain-schedule.yml

```diff
@@ -0,0 +1,16 @@
+name: Build release Swift toolchains
+
+on:
+  workflow_dispatch:
+  # Schedule to build a new release toolchain nightly.
+  schedule:
+    - cron: "10 0 * * */1"
+
+jobs:
+  # Each job builds a release toolchain for a specific Swift version.
+  build-release-5_10:
+    uses: compnerd/swift-build/.github/workflows/swift-toolchain.yml@release/5.10
```

Comment:
Why does it need the `compnerd/swift-build` prefix? Should `.github/workflows/swift-toolchain.yml@release/5.10` not be repository relative?

Reply:
Good question, let me check

Reply:
@compnerd unfortunately GitHub has a limitation for local workflow reference - I can't specify another branch:

```
invalid value workflow reference: cannot specify version when calling local workflows
```

So it looks like I have to use the owner/repo prefix :/

Reply:
I wonder if we can use something from the github context to get the owner/repo (i.e. `${{ github.repository }}` instead of `compnerd/swift-build`.  If not, lets add a comment explaining why this is hardcoded.

Reply:
Unfortunately that doesn't work:
```
invalid value workflow reference: references to workflows must be prefixed with format 'owner/repository/' or './' for local workflows
```

I added a note now in the updated patch.